tox.ini

```diff
@@ -10,6 +10,7 @@ envlist =
     py310-oldestdeps
     build_docs{,-notebooks}
     codestyle
+    benchmarks
```

Comment:
I think we should add a tox env which uses pytest-benchmark to run the benchmarks and displays some output.